pkg/mountutil/volumestore/volumestore.go

```diff
@@ -31,21 +31,33 @@ import (
 	"github.com/containerd/nerdctl/v2/pkg/strutil"
 )
 
-// Path returns a string like `/var/lib/nerdctl/1935db59/volumes/default`.
-func Path(dataStore, ns string) (string, error) {
```

Comment:
This should not be public.
Consumer should only use the interface methods and not fiddle with the underlying (filesystem based) implementation.